fluxc/src/main/java/org/wordpress/android/fluxc/store/mobile/FeatureFlagsStore.kt

```diff
@@ -46,6 +46,12 @@ class FeatureFlagsStore @Inject constructor(
         return featureFlagConfigDao.getFeatureFlagList()
     }
 
+    // This returns a list because there can be multiple values for a single key.
+    // It will be the client's responsibility to decide which value to use.
+    fun getFeatureFlag(key: String): List<FeatureFlag> {
```

Comment:
NP: Since we can get multiple flags back.
```suggestion
    fun getFeatureFlagsByKey(key: String): List<FeatureFlag> {
```

Reply:
Thanks! So much better.